website/docs/releases.md

```diff
@@ -26,6 +26,8 @@ In case you are wondering why all our releases start with `0.0`, read [this FAQ
 
 ### Upcoming Release
 
+* Introduced new [`array` SQL datatypes](./sql/datatype/array.md).
```

Comment:
this seems completely broken...

There should not be an "Upcoming release" section on the main branch, at all. All "upcoming release" release notes should be on the `upcoming` branch. Please figure out why this got messed up and how we need to fix our process

Reply:
It seems to me that this was all done manually and simply forgotten with the previous merge into `main`. We should probably write a GitHub Workflow for this.